### ISSUE TITLE
bug:  ipfshttpclient.exceptions.VersionMismatch: Unsupported daemon version '0.7.0' 

### ISSUE THREAD

**Issue by thelostone-mc** [ipfs-shipyard/py-ipfs-http-client#240]
#### Description 

[ipfshttpclient==0.6.0](https://github.com/gitcoinco/web/blob/master/requirements/ci.txt#L2)
[image: ipfs/go-ipfs:v0.6.0](https://github.com/gitcoinco/web/blob/master/docker-compose.yml#L45
)

CI link: https://travis-ci.org/github/gitcoinco/web/builds/731291982

PR where I'm trying to get it to pass by upgrading ipfshttpclient -> 0.6.1
https://github.com/gitcoinco/web/pull/7561

```
version = [0, 7, 0], minimum = [0, 4, 22], maximum = [0, 7, 0], blacklist = []
    def assert_version(version, minimum=VERSION_MINIMUM, maximum=VERSION_MAXIMUM, blacklist=VERSION_BLACKLIST):
    	"""Make sure that the given daemon version is supported by this client
    	version.
    
    	Raises
    	------
    	~ipfshttpclient.exceptions.VersionMismatch
    
    	Parameters
    	----------
    	version : str
    		The actual version of an IPFS daemon
    	minimum : str
    		The minimal IPFS daemon version to allowed
    	maximum : str
    		The maximum IPFS daemon version to allowed
    	"""
    	# Convert version strings to integer tuples
    	version = list(map(int, version.split('-', 1)[0].split('.')))
    	minimum = list(map(int, minimum.split('-', 1)[0].split('.')))
    	maximum = list(map(int, maximum.split('-', 1)[0].split('.')))
    
    	if minimum > version or version >= maximum:
>   		raise exceptions.VersionMismatch(version, minimum, maximum)
E     ipfshttpclient.exceptions.VersionMismatch: Unsupported daemon version '0.7.0' (not in range: 0.4.22 ≤ … < 0.7.0)
````

**Comment by thelostone-mc** [comment]
Looks similar to https://github.com/ipfs-shipyard/py-ipfs-http-client/issues/239

**Comment by ntninja** [comment]
It's fixed in the 0.7.0a1 prerelease.
